src/components/Avatar/index.tsx

```diff
@@ -31,18 +33,23 @@ export const Avatar = ({
 		<>
 			{src && (
 				<GrommetAvatar
-					size={emphasized ? 'large' : 'medium'}
+					size={emphasized ? '36px' : '24px'}
```

Comment:
Hmm I'm fine with it for now, but maybe we want this to be defined using `fontSize` maybe? Let's see later on, it's fine for now.

Reply:
I thought about it but the `fontSizes` does not contain the `36px` that we would like to have when emphasized. We have 34 instead. Same for the background, we can put a default background for sure and then overwrite it with the `background` prop if necessary. I'll open a new PR, so we can discuss it there 👍